interface/modules/zend_modules/config/modules.config.php

```diff
@@ -0,0 +1,11 @@
+<?php
+return [
+    'Carecoordination',
+    'Ccr',
+    'Documents',
+    'Immunization',
+    'Syndromicsurveillance',
+    'Patientvalidation',
+    'Multipledb',
+    'PrescriptionTemplates'
+];
```

Comment:
Do we need to make this file writable?

Reply:
No, I can actually remove that file.  I was attempting to separate core modules from the modules that are on the installer page, but I took things in a different route.  This was accidentally left in the commit so I'll remove it.